src/lib/message/L1ToL2Message.ts

```diff
@@ -259,6 +283,68 @@ export class L1ToL2MessageReader extends L1ToL2Message {
     return null
   }
 
+  private async getFirstRedeem(
+    retryableCreationReceipt?: TransactionReceipt,
+    includeFail?: boolean
```

Comment:
If we do keep this, we can give it a default value
```suggestion
    includeFail = false
```

instead of
```
    includeFail = includeFail || false
```